app/views/users/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if user_signed_in? %>
+
+<h1>All users</h1>
+
+  <% @users.each do |user| %>
+    <li>
+      <%= link_to user.name, users_index_path %>
```

Comment:
Please explain where this link leads to

Reply:
Along with this: get 'users / index' in routes, it displays a list of all users. You told me to do it
![Screenshot from 2019-08-25 22-34-54](https://user-images.githubusercontent.com/46228563/63654846-99b89000-c788-11e9-9f4e-19034f60acc0.png)